packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingInvoiceDetails.tsx

```diff
@@ -344,7 +344,7 @@ const ExistingInvoiceDetailsBase = (props: ExistingReceivableDetailsProps) => {
               ) : (
                 <FileViewer
                   mimetype="application/pdf"
-                  url={pdf?.file_url}
+                  url={pdf?.file_url ?? ''}
```

Comment:
Sorry, still don't understand why we need to render `FileViewer` if for some reason the link is empty

Reply:
Done